src/attributes/utils/handlers.ts

```diff
@@ -17,6 +26,99 @@ import {
 } from "../types/AttributeValueDelete";
 import { getFileValuesToUploadFromAttributes, isFileValueUnused } from "./data";
 
+export function createAttributeChangeHandler(
+  changeAttributeData: FormsetChange<string[]>,
+  triggerChange: () => void
+): FormsetChange<string> {
+  return (attributeId: string, value: string) => {
+    triggerChange();
+    changeAttributeData(attributeId, value === "" ? [] : [value]);
```

Comment:
maybe instead 
```suggestion
    changeAttributeData(attributeId, !!value ? [] : [value]);
```

Reply:
I just moved code from one place to another and it needs more refactor, as there are cases, which use array null or undefined as array value. I know it seems to be wrong, but I think I'll change it in other PR.